ballerina-tests/tests/62_message_size_service.bal

```diff
@@ -55,13 +55,13 @@ service "HelloWorld62" on msgSizeService {
     }
 
     remote function msgSizeBidiStreaming(HelloWorld62StringCaller caller, stream<string, error?> clientStream) returns error? {
-        check clientStream.forEach(function(string val) {
+        checkpanic clientStream.forEach(function(string val) {
```

Comment:
```suggestion
        check clientStream.forEach(function(string val) {
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/606/commits/c32ef7e98e2bfaa4df966a96025d16d755dfb1cd